### DISCUSSION TITLE
和nextjs一起用报document is not defined

### DISCUSSION THREAD

**Discussion by yizhixiaotaozi** [DouyinFE/semi-design#2379] [Q&A]
和nextjs一起用报document is not defined。
尝试了把nextjs组件写成use client、异步加载组件、指定nextjs使用edge环境都不行。谁知道怎么弄不？

**Comment by yizhixiaotaozi** [comment]
const LoginForm = dynamic(() => import('./_components/LoginForm'),{ ssr: false })  异步导入，禁止ssr可以。 有更好的办法么？

**Reply by bairihai** [reply]
想不出来

**Comment by varichs** [comment]
楼主使用成功了吗
